index.html

```diff
@@ -2,6 +2,7 @@
 <html>
 <head>
 <meta charset="utf-8">
+<link rel="stylesheet" href="./style.css" />
 <title>CacheIR Analyzer</title>
 <style>
 
```

Comment:
Please also move the existing stylesheet to the new file

Reply:
Done :+1: